src/default-xpring-client.ts

```diff
@@ -2,68 +2,19 @@ import { Utils, Wallet } from 'xpring-common-js'
 import bigInt, { BigInteger } from 'big-integer'
 import { XpringClientDecorator } from './xpring-client-decorator'
 import TransactionStatus from './transaction-status'
-import RawTransactionStatus from './raw-transaction-status'
-import GRPCNetworkClient from './grpc-network-client'
+import RawTransaction from './raw-transaction'
+import GRPCNetworkClientNode from './grpc-network-client'
 import GRPCNetworkClientWeb from './grpc-network-client.web'
-import { NetworkClient } from './network-client'
-import { GetTxResponse } from './generated/web/rpc/v1/tx_pb'
-import { GetFeeResponse } from './generated/web/rpc/v1/fee_pb'
-import { XRPDropsAmount } from './generated/web/rpc/v1/amount_pb'
+import { XRPDropsAmount } from './generated/web/rpc/v1/amount_pb' // TODO would be nice to have 0 references to generated files here
```

Comment:
Humbly noted.  Want to add this as an actionable todo and format it with an owner:

```
// TODO(keefertaylor@): Do X which removes XRPDropsAmount dependency.
```